CONTRIBUTING.md

```diff
@@ -50,3 +50,55 @@ To test a beta release of `@netlify/build` in a site on Netlify, set the environ
 `NETLIFY_BUILD_CLI_VERSION` to the NPM tag you wish to use.
 
 After submitting the pull request, please make sure the Continuous Integration checks (GitHub actions) are passing.
+
+## Releasing
+
+To release a specific package:
+
+```bash
+# On the root directory.
+# Do not release if linting or tests are failing. This might indicate a bug.
+$ npm test
+
+# Go inside the package directory
+$ cd packages/packageName
+
+# Replace "patch" with "minor" or "major" or "prerelease" depending on the case
+$ npm version patch
+$ git add -p
+$ git commit
+
+$ npm publish
+```
+
+If a package is released, the other packages that depend on it should increment their version of that package inside
```

Comment:
Looks good.
Is the part of releasing dependant packages manual at the moment?
We should probably automate that if not.

Reply:
Yes, it is quite manual. We should definitely automate this.